modules/nodes-cluster-worker-latency-profiles-using-at-creation.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * scaling-worker-latency-profiles.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nodes-cluster-worker-latency-profiles-using-at-creation_{context}"]
+= Implementing worker latency profiles at cluster creation
+
+The `workerLatencyProfile` must be added to the manifest in the following sequence:
+
+. Create the manifest needed to build the cluster.
+. Edit the `node.config` object to add the name of the profile. You can change the profile at any time as latency increases or decreases.
+. When defining `workerLatencyProfile` in the manifest for the first time, give it the value "Default".
```

Comment:
Here, you can specify any of the profiles at cluster creation time (Default, MediumUpdateAverageReaction or LowUpdateSlowReaction)

Reply:
OH! OK! Thank you!